package.json

```diff
@@ -57,8 +57,8 @@
     "protractor": "~5.1.2",
     "request-promise": "^4.2.1",
     "ts-node": "~3.0.4",
-    "tslint": "~5.3.2",
+    "tslint": "^5.8.0",
```

Comment:
Is there a particular reason you are updating this?  If so I think we should include this change in #184 where we are making more changes to the `package.json`